core/src/main/java/com/onelogin/saml2/Auth.java

```diff
@@ -78,6 +79,16 @@
 	private DateTime sessionExpiration;
 
 	/**
+	 * The ID of the last assertion processed
+	 */
+	private String assertionId;
```

Comment:
In order to be align with the lastRequestId parameter and getLastRequestId method:

can we rename?:
- the assertionId and assertionNotOnOrAfter parameters by lastAssertionId and lastAssertionNotOnOrAfter.
- the getAssertionId and getAssertionNotOnOrAfter methods by getLastAssertionId and getLastAssertionNotOnOrAfter.